pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "netutils"
-version = "1.13.0"
+version = "1.14.0"
```

Comment:
Can you remove this from your PR? We'll make a v1.14.0 prep PR where we'll bump the version and add any release notes since there will (most likely) be other updates.